catroid/src/org/catrobat/catroid/formulaeditor/UserVariable.java

```diff
@@ -55,4 +56,12 @@ public String getName() {
 	public void setName(String name) {
 		this.name = name;
 	}
+
+	public boolean getVisibility() {
```

Comment:
`isVisible()`
Consider to rename visibility to visible as well because of: `if (visible)` vs `if (visibility)`